src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -257,6 +257,8 @@ public class AdminController : Controller
             // Load items so that loading handlers are invoked.
             var pageOfContentItems = await query.Skip(pager.GetStartIndex()).Take(pager.PageSize).ListAsync(_contentManager);
 
+            _httpContextAccessor.HttpContext.Features.Set(new ContentItemListFeature(pageOfContentItems));
```

Comment:
This is not really what Features are meant for.

Reply:
Alternatively, we can invoke an event that would let's other hook into it. Do you have a better suggestion?